service/s3/api_client.go

```diff
@@ -370,6 +386,21 @@ func getBucketFromInput(input interface{}) (*string, bool) {
 	}
 }
 
+// supportAccelerate returns a boolean indicating if the operation associated with
+// the provided input supports S3 Transfer Acceleration
+func supportAccelerate(input interface{}) bool {
```

Comment:
Does this include the fixes that were made to v1 SDK around some operations were not decorated as supporting accelerate and outposts?

Reply:
Nothing related to outposts. `supportAccelerate` func tells you if accelerate is allowed for a particular operation. So yes, handles some operations that do not support accelerate.